src/Module/Api/Mastodon/PushSubscription.php

```diff
@@ -94,37 +112,45 @@ public function put(array $request = [])
 
 		$ret = Subscription::update($application['id'], $uid, $fields);
 
-		Logger::info('Subscription updated', ['result' => $ret, 'application-id' => $application['id'], 'uid' => $uid, 'fields' => $fields]);
+		$this->logger->info('Subscription updated', [
+			'result' => $ret, 'application-id' => $application['id'],
+			'uid'    => $uid, 'fields' => $fields
```

Comment:
If you're going for readability, please use the full length declaration. In this context, please add a trailing comma on each line to streamline diffs.
```suggestion
			'result'         => $ret,
			'application-id' => $application['id'],
			'uid'            => $uid,
			'fields'         => $fields,
```